docs/gettsim_objects/input_variables.md

```diff
@@ -555,6 +555,13 @@ implemented in 2006 (§ 237 SGB VI Abs. 5).
 
 Type: bool
 
+(höchster_bruttolohn_letzte_15_jahre_y)=
+
+## `höchster_bruttolohn_letzte_15_jahre_y`
```

Comment:
Just curious: Say I'm early retired at 61 and my highest wage was at 46. What happens the next year? Does wage at 46 still count or does it drop? Put differently, is it "the 15 years before the first pension claim" or "the past 15 years from today"?

Please clarify in name and docstring.

Reply:
Good catch, it's 15 years before pension claiming.